webpack.config.js

```diff
@@ -18,7 +18,7 @@ const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 // Common module rules shared between storybook and production builds.
 const rules = [];
 
-if (process.env.NODE_ENV === 'development') {
+if (process.env.NODE_ENV === 'development' || process.env.STORYBOOK_BUILD) {
```

Comment:
You'll prob want to split off your storybook config to act like a consumer at some point. It'll make it more obvious if there's something up with the library.

Reply:
yes, absolutely. I put a note about this in the docs because it is just too much to an already large PR, but it does create a problem because tsdx transforms sometimes don't match webpack transpilation from storybook which can lead to build bugs that aren't present in storybook preview. Big pain but hopefully short-lived.